### DISCUSSION TITLE
How to distinguish UFBoot and SH-aLRT values in an output?

### DISCUSSION THREAD

**Discussion by finwe858** [iqtree/iqtree2#249] [Q&A] [answered]
Hi,
I run a ML analyses on the IQ web server with the next command line:
../../iqtree -s cyt-134-haploonly+corrected.fas -st DNA -m HKY+I+G4+F -bb 1000 -alrt 1000
In the output trees both balues are provided with a /, but I cannot find what value stands for what analyses. I assumed that UB should be the first number (as it is first in the command line)? In my output most values before '/' are lower than after it. 97.2/100, 43.1/89, 73.3/96 as few exmples
In the FAQ it is recommended to:  One would typically start to rely on the clade if its SH-aLRT >= 80% and UFboot >= 95%.
So, still confused what values stand for the UB and which for SH.
Thanks

**Comment by roblanf** [comment]
@bqminh will know the answer to your actual question, but I just want to chime in here to register my (strong) disagreement with the FAQs here. In my opinion there is no threshold above which one can 'rely' on a clade.

UFBOOT is a measure of sampling variance (as is any bootstrap in phylogenetics), that's it. So datasets with more information (e.g. longer alignments) will have higher bootstrap values *regardless* of whether a clade is reliable. The aLRT should behave similarly, but for different reasons (it's a measure of whether a branch length is distinguishable from zero). This is covered extensively in the literature, but my favourite paper on it is this one:

https://academic.oup.com/mbe/article/29/2/457/1024815

On that note, I think we should remove these mentions of arbitrary thresholds from the FAQs, but I'll open another discussion for that. (And apologies for not actually answering the question!)

**Reply by roblanf** [reply]
OK I started a separate discussion on this point here: https://github.com/iqtree/iqtree2/discussions/250.

Anyone with opinions (backed up with evidence) please join that discussion!

**Comment by roblanf** [comment, marked as answer]
The answers: 

* the label `73.3/96` is `aLRT/UFBOOT`
* the order you put stuff on the command line doesn't make any difference

Here's how you can figure it out (and how I figured it out...). If you take a small example file you can run command lines to distinguish what's going on, including whether changing the order on the command line changes the output. I increased the sample sizes just to get more reproducible estimates (small differences are still expected, because I didn't fix the random number seed): 

```
iqtree2 -s example.phy -st DNA -m JC -bb 10000 -alrt 10000  -pre bb_alrt # alrt first
iqtree2 -s example.phy -st DNA -m JC -alrt 10000 -bb 10000  -pre alrt_bb # bb first
iqtree2 -s example.phy -st DNA -m JC -bb 10000 -pre bb  
iqtree2 -s example.phy -st DNA -m JC -alrt 10000 -pre alrt 
```

Now you can look at the output trees by:

```
more bb_alrt.treefile 
more alrt_bb.treefile 
more bb.treefile
more alrt.treefile
```

Without printing all the trees, I'll just focus on one node where the two values were quite different, so easy to distinguish (i put in the branch length so you can see it's the same branch!):

| tree | branch label |
|------|-------------|
|  bb_alrt | `39/56:0.0226144245` |
|  alrt_bb | `38.9/57:0.0226144245` |
|  bb | `56:0.0226144245` |
| alrt | `37.5:0.0226144245` |

Hope that helps!

R

**Reply by bqminh** [reply]
If you open .iqtree file, you will see this line:

```
Numbers in parentheses are SH-aLRT support (%) / ultrafast bootstrap support (%)
```

So the first number is SH-aLRT and the 2nd number is UFBoot. It doesn't matter whether you have -bb or -alrt options first. This order will stay the same.
